src/treq/client.py

```diff
@@ -39,6 +39,7 @@
 
 
 _NOTHING = object()
+CONTENT_TYPE_HEADER_NAME = b'Content-Type'
```

Comment:
This is a personal design decision to reduce typos. Please let me know if it runs counter to the project's intended design.